tests/codex/node/testcontracts.nim

```diff
@@ -91,6 +91,8 @@ asyncchecksuite "Test Node - Host contracts":
 
     protected = (await erasure.encode(manifest, 3, 2)).tryGet()
     builder = Poseidon2Builder.new(localStore, protected).tryGet()
+    (await builder.init()).tryGet()
```

Comment:
Hm... this looks quite weird (doing a get on a void) but OK, I don't have better ideas. :-)

Reply:
Yeah, it's odd... but it's a way to force an exception in case there is no value.

Another way (not sure if it's "better"):
```nim
if (await builder.init()).isErr: fail()
```
But this doesn't give much output if it does fail, so an even more explicit way is:
```nim
if error =? (await builder.init()).errorOption:
  echo error.msg
  fail()
```

Reply:
(await ...).tryGet() calls are all over the testing code. It fails the test when the result is an error. How would you like to see this changed?